README.md

```diff
@@ -1,49 +1,52 @@
 # Getting started with `appmap-agent-js`
 
-`appmap-agent-js` is a JavaScript recording agent for the [AppMap](https://appland.org) framework.
-
+`appmap-agent-js` is a JavaScript recording agent for the [AppMap](https://appmap.io) framework.
 
 ## Introduction
  
-`appmap-agent-js` records AppMaps from Node.js processes when they run. There are two recommended strategies for recording AppMaps when getting started:
+`appmap-agent-js` records AppMaps from Node.js processes when they run. There are several strategies for recording AppMaps:
 
-1. Record `mocha` test cases when they run.
+1. Record test cases. At the moment, we support [jest](https://jestjs.io) and [mocha](https://mochajs.org).
 2. Record Node.js processes using start/stop controls via http calls to web endpoints. These are implemented by the AppMap agent to let users control the recording remotely.
-
-`appmap-agent-js` starts the test and application processes that will be recorded.
-
+3. Record Node.js processes from start to finish.
 
 ## Installation and setup
 
 ### Requirements
 
 Supported platforms:
-* Mainstream os: linux distros, macOS, and windows
-* Node.js: 14, 16, 17 (end-of-life), or 18
-* Express.js: 4
-* git is highly recommended
-* mocha >= 8.0.0 is required for recording AppMaps from test cases (earlier versions do not support required root hooks)
+* Mainstream os: linux distros, macOS, and windows.
```

Comment:
You might capitalize Linux and Windows while you're at it :)

Reply:
rebased in 451694174c2afbe308f56f80f666faca3864677b